mypyc/primitives/generic_ops.py

```diff
@@ -334,3 +334,19 @@
     c_function_name="CPyIter_Next",
     error_kind=ERR_NEVER,
 )
+
+# this would be aiter(obj) if it existed
+aiter_op = custom_op(
+    arg_types=[object_rprimitive],
+    return_type=object_rprimitive,
+    c_function_name="CPy_GetAIter",
+    error_kind=ERR_MAGIC,
+)
+
+# this would be anext(obj) if it existed
+anext_op = custom_op(
```

Comment:
Since you are adding primitives, maybe also add few IR tests?

Reply:
I'm not sure if it would add much value here?

Reply:
(I went and merged but will add IR tests if you think it's important)

Reply:
They are much faster than run tests, and also I remember similar tests for non-async for were helpful when I implemented some special-casing a while ago (but maybe I care too much about generated code looking good, not just being correct).